Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -438,7 +484,7 @@ - (nullable UIView*)tableView:(UITableView*)tableView viewForFooterInSection:(NS
 - (void)tableView:(UITableView*)tableView willDisplayCell:(UITableViewCell*)cell forRowAtIndexPath:(NSIndexPath*)indexPath {
     id<WMFExploreSectionController> controller = [self sectionControllerForSectionAtIndex:indexPath.section];
 
-    if ([controller respondsToSelector:@selector(willDisplaySection)] && (![self isDisplayingCellsForSectionController:controller] || [self rowAtIndexPathIsOnlyRowVisibleInSection:indexPath])) {
```

Comment:
@bgerstle i guess this goes with my other comment - why are we removing the check here? 

That would seem to suggestion that the call to `sendWillDisplayToVisibleSectionControllers` when the app returns to the foreground isn't working.